app/views/pages/course-no-classes.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="headline-poster-wrapper">
+  <h1 class="headline-poster headline-poster-top-billing narrow-content">
+    Course 16
+  </h1>
+  <h2 class="headline-poster headline-poster-copy narrow-content">
+    Aerospace Engineering doesn’t have any classes yet
+  </h2>
+</div>
+
+<%= link_to "Add a class", "#!", class: "headline-poster headline-poster-action" %>
+
+<script type="text/javascript">
+  $(function() {
+    $(".headline-poster-action").mouseenter(function(){
+      $("body").addClass('headline-poster-takeover');
+    });
+    $(".headline-poster-action").mouseleave(function(){
```

Comment:
You won't notice this much in this case above, but it's best to cache selectors (this is so lightweight though):

`var $headlinePosterAction = $(".headline-poster-action");`